features/comment.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress comments through the REST API
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+    When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
I disagree. The test cases are separate from travis. And separate from WP installs. The cases a behat and run separately (like I've done forever since I started using these test tools).
Behat docs suggest that the background givens ensure that the environment is setup as expected. It is not guaranteed (since these behat cases are separate) that WP is any specific version. Therefore, the test for the specific version.
Since this only makes the test cases stronger, I can't understand why you would not allow this to remain. It is not enough to just have a WP install. It must be a WP 4.7+ install.